vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/ScriptSteps.java

```diff
@@ -71,10 +81,14 @@ public WebElement thenJavascriptFileWithNameIsIncludedInTheSourceCode(String jsF
      *
      * @param jsText Content of the <i>{@literal <script>}</i> tag.
      * @return webElement found js script
+     * @deprecated Use step:
+     * This step "Then a javascript with the text '$jsText' is included in the source code" will be removed
      */
+    @Deprecated(since = "0.6.5", forRemoval = true)
     @Then("a javascript with the text '$jsText' is included in the source code")
     public WebElement thenJavascriptFileWithTextIsIncludedInTheSourceCode(String jsText)
     {
+        LOGGER.warn(String.format(DEPRECATED_STEP_MESSAGE, "thenJavascriptFileWithTextIsIncludedInTheSourceCode"));
```

Comment:
😶‍🌫️  we need to provide the step syntax, not method name; also it makes sense to provide recommendation which step can be potentially used as a replacement